helpdesk/teams/tables.py

```diff
@@ -13,3 +13,23 @@ class Meta:
         model = Team
         exclude = ("id", "pit_location")
         order_by = "tla"
+
+
+class TeamAttendanceTable(tables.Table):
+    name = tables.LinkColumn("teams:team_detail", args=[tables.A("tla")])
+    latest_event__0__type = tables.Column("Latest Event")
+    latest_event__0__comment = tables.Column("Comment")
+    latest_event__0__created_at = tables.DateTimeColumn(verbose_name="Time")
+    user = tables.TemplateColumn(
+        verbose_name="Logged by",
+        template_code='{{record.latest_event.0.user|default:"—"}}',
+    )
+    actions = tables.LinkColumn("teams:team_log_attendance_form", args=[tables.A("tla")], text="Log")
+
+    def render_latest_event__0__type(self, value: str) -> str | None:
+        lookups = dict(TeamAttendanceEventType.choices)
+        return lookups.get(value)
```

Comment:
Suggestion: Because this is an enum, I think you can do:

```suggestion
        return TeamAttendanceEventType[value].value
```

Reply:
This didn't work